src/components/pages/devices/flyouts/deviceDetails/deviceDetails.js

```diff
@@ -266,12 +266,17 @@ export class DeviceDetails extends Component {
                         </GridHeader>
                         <GridBody>
                           {
-                            properties.map(([propertyName, propertyValue], idx) =>
-                              <Row key={idx}>
-                                <Cell className="col-3">{propertyName}</Cell>
-                                <Cell className="col-7">{propertyValue.toString().replace(/,/g, ", ")}</Cell>
-                              </Row>
-                            )
+                            properties.map(([propertyName, propertyValue], idx) => {
+                              const displayValue = !device.desiredProperties[propertyName] || propertyValue === device.desiredProperties[propertyName]
+                                ? propertyValue
+                                : `${propertyValue} (${t('devices.flyouts.details.properties.syncing')} ${device.desiredProperties[propertyName]})`;
```

Comment:
This string should probably take variables for the values. Something "{{reportedPropertyValue}} syncing {{desiredPropertyValue}}" to account for potential language order issues.